tensorflow_datasets/image/__init__.py

```diff
@@ -38,3 +38,5 @@
 from tensorflow_datasets.image.quickdraw import QuickdrawBitmap
 from tensorflow_datasets.image.rock_paper_scissors import RockPaperScissors
 from tensorflow_datasets.image.svhn import SvhnCropped
+from tensorflow_datasets.image.cycle_gan import CycleGANConfig 
```

Comment:
this line is probably not needed.

Reply:
Ok, removing this line produces no error, but can you please tell me why it is used [here](https://github.com/tensorflow/datasets/blob/96fe8c3156b43eb776456522ac739a0a6cdb86d2/tensorflow_datasets/text/__init__.py#L19), [here](https://github.com/tensorflow/datasets/blob/96fe8c3156b43eb776456522ac739a0a6cdb86d2/tensorflow_datasets/text/__init__.py#L21), [here](https://github.com/tensorflow/datasets/blob/96fe8c3156b43eb776456522ac739a0a6cdb86d2/tensorflow_datasets/audio/__init__.py#L19) and [here](https://github.com/tensorflow/datasets/blob/96fe8c3156b43eb776456522ac739a0a6cdb86d2/tensorflow_datasets/video/__init__.py#L21). Should I modify these also ?

Reply:
you're right. My bad.